src/helpers/resolverHelpers.js

```diff
@@ -1,30 +1,20 @@
 import hash from 'string-hash'
 
 export const getUniqueAuthors = (artworks) => {
-  let uniqueAuthors = {}
-  artworks.map((artwork) => {
-    artwork.authors[0].map((author) => {
-      const id = author.author
-      if (!uniqueAuthors[id]) {
-        uniqueAuthors[id] = author
-        // uniqueAuthors[id].artworks = []
-        uniqueAuthors[id].mediums = []
-      }
-
-      // if (uniqueAuthors[id].artworks.indexOf(artwork.id) < 0)
-      //   uniqueAuthors[id].artworks.push(artwork.id)
-
-      if (artwork.mediums && artwork.mediums[0].text) {
-        const mediumId = hash(artwork.mediums[0].text)
-        if (uniqueAuthors[id].mediums.indexOf(mediumId) < 0)
-          uniqueAuthors[id].mediums.push(mediumId)
-      }
+  const authors = artworks.map(artwork => {
+    return artwork.authors[0].map(author => {
+      return JSON.stringify({
```

Comment:
`JSON.stringify` is our hash function TT, ordering matters TT, this is supposedly a hard problem.

However we are working in the realm of maybe like, 100,000 items max. So maybe its fine? Maybe caching will fix everything?